.github/workflows/upload-test-stats.yml

```diff
@@ -2,7 +2,7 @@ name: Upload test stats
 
 on:
   workflow_run:
-    workflows: [pull, trunk, periodic, inductor, inductor-A100-perf]
```

Comment:
You don't need `inductor-A100-perf-nightly` both in `upload-test-stats.yml` and in `upload-torch-dynamo-perf-stats`.  Once you have the commit in https://github.com/pytorch/pytorch/pull/96165, you will just need to update the latter `upload-torch-dynamo-perf-stats.yml`